main.js

```diff
@@ -0,0 +1,68 @@
+let billAmountInput = document.querySelector('.bill-amount');
+let numOfPeopleInput = document.querySelector('.num-of-people');
+let calcBtn = document.querySelector('.calc-btn');
+let clearBtn = document.querySelector('.clear-btn');
+
+window.addEventListener('load', handlePageLoad);
+billAmountInput.addEventListener('keyup', btnStatus);
+numOfPeopleInput.addEventListener('keyup', btnStatus);
+calcBtn.addEventListener('click', calculateTotal);
+clearBtn.addEventListener('click', resetInputs);
+
+function handlePageLoad() {
+  btnStatus();
+}
+
+function btnStatus() {
+  let billAmount = parseFloat(billAmountInput.value);
+  let numOfPeople = parseFloat(numOfPeopleInput.value);
+  if ( billAmount > 0 && numOfPeople > 0) {
+    calcBtn.disabled = false;
+    clearBtn.disabled = false;
+  } else {
+    calcBtn.disabled = true;
+    clearBtn.disabled = true;
+  }
+}
+
+function calculateTotal() {
+  let serviceRating = document.querySelector('select').value;
+  let billAmount = billAmountInput.value;
+  let numOfPeople = numOfPeopleInput.value;
+
+  let tipAmount = (billAmount * serviceRating).toFixed(2);
+  tipAmount = parseFloat(tipAmount);
+
+  let splitBillAmount = (billAmount / numOfPeople).toFixed(2);
```

Comment:
Dividing and using `toFixed(2)` might leave off a penny if the decimal value is odd. It might be better to divide the amount and keep subtracting it from the total until there's a penny left, then add it to the last split amount.